http/server.lua

```diff
@@ -1321,6 +1384,56 @@ local function httpd_start(self)
     return self
 end
 
+local function validate_ssl_opts(ssl_opts)
+    if ssl_opts.ssl_cert_file ~= nil then
+        if type(ssl_opts.ssl_cert_file) ~= 'string' then
+            error("ssl_cert_file option must be a string")
+        end
+        if fio.path.exists(ssl_opts.ssl_cert_file) ~= true then
+            errorf("file %q not exists", ssl_opts.ssl_cert_file)
+        end
+    end
+
+    if ssl_opts.ssl_key_file ~= nil then
+        if type(ssl_opts.ssl_key_file) ~= 'string' then
+            error("ssl_key_file option must be a string")
+        end
+        if fio.path.exists(ssl_opts.ssl_key_file) ~= true then
+            errorf("file %q not exists", ssl_opts.ssl_key_file)
+        end
+    end
```

Comment:
ssl_cert_file and ssl_key_file are required for a server, not optional.

Reply:
Since we don't pass boolean `enable_ctx` option (according one of the previous comment), checks on `nil` is required to determine whether `tls` is enabled.